packages/dsh-tauri-pet/package.json

```
@@ -0,0 +1,73 @@
+    "start": "tsx src/index.ts",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "dsh-tauri": "workspace:*"
+  },
+  "devDependencies": {
+    "@deepseek-ai/dsh-client-store": "catalog:dsh",
+    "@deepseek-ai/dsh-client-ui-layout": "catalog:dsh",
+    "@deepseek-ai/dsh-client-ui-renderer": "catalog:dsh",
+    "@deepseek-ai/dsh-client-ui-slots": "catalog:dsh",
+    "@deepseek-ai/cordis": "catalog:dsh",
```

ESLint

[error] Expected object keys to be in ascending order. '`@deepseek-ai/cordis`' should be before '`@deepseek-ai/dsh-client-store`'.

(jsonc/sort-keys)